introduction.md

```diff
@@ -0,0 +1 @@
+hidashjldjkhalskjhglakjhr ajskflhkj asdhjfklk ajhdklfjh lasdjkfh i don't like this part of training
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.